picard/util/tags.py

```diff
@@ -98,6 +177,45 @@ def script_name(self):
         else:
             return self._name
 
+    def notes(self):
+        for attrib, note in ATTRIB2NOTE.items():
+            if getattr(self, attrib):
+                yield note
+
+    def related_options_titles(self):
+        if not self.related_options:
+            return None
+        for setting in self.related_options:
+            title = get_option_title(setting)
+            if title:
+                yield _(title)
+
+    def links(self):
+        if not self.doc_links:
+            return None
+        for doclink in self.doc_links:
+            yield f"<a href='{doclink.link}'>{doclink.title}</a>"
+
+    def see_alsos(self):
+        if not self.see_also:
+            return None
```

Comment:
Actually a simple `return` can do.

Actually that's equivalent, since returning None is the default for Python functions.

```python
>>> def f():
...     return
... 
>>> f() is None
True
```

```suggestion
            return
```